tests/tools/oesign/test-sign/CMakeLists.txt

```diff
@@ -203,3 +203,65 @@ add_test(
 
 set_tests_properties(tests/oesign-sign-lowercase-debug-property
                      PROPERTIES PASS_REGULAR_EXPRESSION "unknown setting")
+
+# Test valid kss properties argument
+set(OESIGN_SIGN_VALID_KSS
+    "[-c,${OESIGN_TEST_INPUTS_DIR}/kss_valid.conf,-k,${OESIGN_TEST_INPUTS_DIR}/sign_key.private.pem]"
+)
```

Comment:
Since this directory does not have 'sgx' in the title, I would expect there to be some "if sgx" check before settings such things.

Reply:
Related code move in block when KSS is enabled so setting will not applied if KSS not supported